platform/linux-generic/odp_traffic_mngr.c

```diff
@@ -4685,10 +4685,10 @@ void odp_tm_stats_print(odp_tm_t odp_tm)
 
 	ODP_PRINT("odp_tm_stats_print - tm_system=0x%" PRIX64 " tm_idx=%u\n",
 		  odp_tm, tm_system->tm_idx);
-	ODP_PRINT("  input_work_queue size=%u current cnt=%u peak cnt=%u\n",
-		  INPUT_WORK_RING_SIZE, input_work_queue->queue_cnt,
+	ODP_PRINT("  input_work_queue size=%u current cnt=%" PRIu64 " peak cnt=%u\n",
+		  INPUT_WORK_RING_SIZE, odp_atomic_load_u64(&input_work_queue->queue_cnt),
```

Comment:
I think the format for peak_cnt should also be fixed to use PRIu32.

Reply:
Fixed in v2.